monai/deploy/operators/dicom_series_to_volume_operator.py

```diff
@@ -90,6 +95,35 @@ def generate_voxel_data(self, series):
         vol_data = np.stack([s.get_pixel_array() for s in slices], axis=0)
         vol_data = vol_data.astype(np.int16)
 
+        # For now we support monochrome image only, for which DICOM Photometric Interpretation
+        # (0028,0004) has defined terms, MONOCHROME1 and MONOCHROME2, with the former being:
+        #   Pixel data represent a single monochrome image plane. The minimum sample value is
+        #   intended to be displayed as white after any VOI gray scale transformations have been
+        #   performed. See PS3.4. This value may be used only when Samples per Pixel (0028,0002)
+        #   has a value of 1. May be used for pixel data in a Native (uncompressed) or Encapsulated
+        #   (compressed) format; see Section 8.2 in PS3.5.
+        # and for the latter "The minimum sample value is intended to be displayed as black"
+        #
+        # In this function, pixel data will be interpreted as if MONOCHROME2, hence inverting
+        # MONOCHROME1 for the final voxel data.
+
+        photometric_interpretation = (
+            slices[0].get_native_sop_instance().get("PhotometricInterpretation", "").strip().upper()
+        )
+        presentation_lut_shape = slices[0].get_native_sop_instance().get("PresentationLUTShape", "").strip().upper()
+
+        if not photometric_interpretation:
+            logging.warning("Cannot get value of attribute Photometric Interpretation.")
+
+        if photometric_interpretation != "MONOCHROME2":
+            if photometric_interpretation == "MONOCHROME1" or presentation_lut_shape == "INVERSE":
+                logging.debug("Applying INVERSE transformation as required for MONOCHROME1 image.")
+                vol_data = vol_data * -1 + np.max(vol_data)
```

Comment:
```suggestion
                vol_data = np.amax(vol_data) - vol_data
```
In a recent project I used this code for inversions. Probably without a difference in performance.

Reply:
Thank you. Updated per comment.